apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -538,11 +533,19 @@ static void utc_audio_pcm_set_config_tc_n(void)
 static void utc_audio_pcm_frames_to_bytes_p(void)
 {
 	ssize_t size;
```

Comment:
This function always assumes g_pcm is the only handler and does all the operations on it. This  might not be the case always. Please pass the handle as parameter to this function and do operations on it. For current scenario it might work but later when we extend the framework this will fail